.github/workflows/regression-tests.yml

```diff
@@ -42,12 +49,29 @@ jobs:
         git clone https://github.com/steveicarus/iverilog.git -b "${{matrix.sim-version}}"
         cd iverilog
         bash ./autoconf.sh
-        ./configure
+        bash ./configure --host=x86_64-w64-mingw32 --prefix=/c/iverilog
         make -j $(nproc)
         make install
+        echo "::add-path::C:\iverilog\bin"
     - name: Install Python testing dependencies
       run: |
-        pip install tox tox-gh-actions
-    - name: Test with tox
+        pip install coverage xunitparser pytest pytest-cov
+    - name: Install cocotb
       run: |
-        tox -e py3_mingw
+        python -m pip install  --global-option build_ext --global-option --compiler=mingw32 -e .
+    - name: Test
+      continue-on-error: ${{matrix.may_fail || false}}
+      run: |
+        $pass = $true
+        pytest
+        $pass = $pass -and $?
+        make test
+        $pass = $pass -and $?
+        exit -not $pass
```

Comment:
This might be silly. If one command fails does the step mark failure? Does it keep going and mark failure? Or is it only the exit code of the last command?

Reply:
Nope, it seemingly only considers the outcome of the final command.